### PR TITLE
start testing on 3.13rc

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
a remake of https://github.com/psf/requests/pull/6720 with the new pytest-httpbin

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 -e .[socks]
 pytest>=2.8.0,<9
 pytest-cov
-pytest-httpbin==2.0.0
+pytest-httpbin==2.1.0rc1
```
Comment (graingert) [inline requirements-dev.txt]:
```suggestion
pytest-httpbin==2.1.0
```

File: setup.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "certifi>=2017.4.17",
 ]
 test_requirements = [
-    "pytest-httpbin==2.0.0",
+    "pytest-httpbin==2.1.0rc1",
```
Comment (graingert) [inline setup.py]:
```suggestion
    "pytest-httpbin==2.1.0",
```

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -31,6 +31,7 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
         cache: 'pip'
+        allow-prereleases: true
```
Comment (nateprewitt) [inline .github/workflows/run-tests.yml:34]: Do we still need this or can it be removed now that `pytest-httpbin` is fully released?

Comment (graingert) [reply .github/workflows/run-tests.yml:34]: This is needed for 3.13rc. it doesn't effect pip installs

**Comment by graingert** [comment]
I should probably update the trove classifier

**Review by nateprewitt — CHANGES_REQUESTED** [review]
I think we're also missing an update for the [`tox`](https://github.com/psf/requests/blob/f12ccbef6d6b95564da8d22e280d28c39d53f0e9/tox.ini#L2) file.

**Review by nateprewitt — APPROVED** [review]

**Review by sigmavirus24 — APPROVED** [review]

**Pull request merged**